web/styles/zulip.css

```diff
@@ -2159,8 +2159,13 @@ td.pointer {
 
 .sender_name_hovered {
     .sender_name {
+        cursor: pointer;
         color: hsl(200deg 100% 40%);
     }
+
+    .message-avatar {
+        cursor: pointer;
+    }
```

Comment:
Why are these scoped inside the `sender_name_hovered` class?

Reply:
The avatar and username both belong to `sender_info_hover` class. This implies that hovering over either the avatar or the name will alter the color of the `.sender_name`. Currently, when hovering over either element, the class `sender_name_hovered` is added to username but no such thing is happening for avatar. Would it be better to use add class for avatar when hovered avatar in the code block `$("#main_div").on("mouseover", ".sender_info_hover", function () {
        const $row = $(this).closest(".message_row");
        $row.addClass("sender_name_hovered");
    });`  and then make avatar related changes in `sender_avatar_hovered`?

the file path is web/src/message_list_hover.js